easybuild/easyblocks/g/gromacs.py

```diff
@@ -208,6 +209,15 @@ def configure_step(self):
             # version of GROMACS. Just prepare first part of cmd here
             plumed_cmd = "plumed-patch -p -e %s" % engine
 
+        # Ensure that the GROMACS log files report how the code was patched
+        # during the build, so that any problems are easier to diagnose.
+        if '-DGMX_VERSION_STRING_OF_FORK=' not in self.cfg['configopts']:
```

Comment:
@mabraham I would prefer to only use `-DGMX_VERSION_STRING_OF_FORK` if the GROMACS version actually supports it. It only avoids a warning, but it prevents questions/confusing for people who see the warning popping up in the build log.

Something like this (with the correct version, of course):

```python
        if '-DGMX_VERSION_STRING_OF_FORK=' not in self.cfg['configopts']:
            # only GROMACS >= 2019 supports specifying a version string
            if LooseVersion(self.version) >= LooseVersion('2019'):
                ...
```

Reply:
I'd switch those two if-stmts around, first check for gromacs version

Reply:
Yeah, both work, doesn't matter much. It's perhaps clearer to do the version check first.

Reply:
Done